app/survey_config/census_config.py

```diff
@@ -57,12 +57,17 @@ class CensusSurveyConfig(
         ],
         compare=False,
     )
-    data_layer: Iterable[Mapping] = field(
-        default_factory=lambda: [{"nisra": False}], compare=False
-    )
+
     survey_title: LazyString = lazy_gettext("Census 2021")
     sign_out_button_text: str = lazy_gettext("Save and complete later")
 
+    def get_data_layer(  # pylint: disable=unused-argument, no-self-use
```

Comment:
Could use `_` for unused arg.

Reply:
Ok, unused argument error gone now but still needed `no-self-use` ignore in one place

Reply:
After refactor it seems that the var is always used so removing `_`, cheers.